source/code/utilities/chinese/chinese_translations.hcp

```diff
@@ -87,8 +87,8 @@ class
     {
         Translation_Entry x;
         x.english = L"see you later/in a while";
-        x.pinyin = L"yī huǐ er jiàn";
-        x.chinese = L"一会儿见";
+        x.pinyin = L"dài huǐ er jiàn";
+        x.chinese = L"待會兒見";
```

Comment:
thanks for this. its hard to know if what I get online is china-like and less taiwanese mandarin-- if that even makes sense?  It could equally just be incorrect; I dunno haha